examples/jsm/math/MeshSurfaceSampler.js

```diff
@@ -200,6 +224,29 @@ class MeshSurfaceSampler {
 			targetColor.g = _color.y;
 			targetColor.b = _color.z;
 
+		} else if ( targetColor !== undefined && this.uvAttribute !== undefined ) {
```

Comment:
I don't think this should be either-or with a vertex color result. It can valuable to return both a vertex color in addition to a uv value.